kala.py

```diff
@@ -38,12 +38,17 @@ def get(mongodb, collection):
     # takes a list of tuples for the sort order.
     sort = [tuple(field) for field in sort] if sort else None
 
-    cursor = mongodb[collection].find(
+    results = mongodb[collection].find(
```

Comment:
You're right of course.  It was because at one point I had `cursor = cursor.distinct(distinct)`, which then meant it was no longer a cursor.  Now that I don't have that line, I can revert it.